src/test/java/org/openlmis/report/service/JasperTemplateServiceTest.java

```diff
@@ -235,6 +249,38 @@ public void shouldThrowErrorIfTemplateNameAlreadyExists() throws Exception {
   }
 
   @Test
+  public void shouldThrowErrorIfImageDoesNotExist() throws Exception {
+    expectedException.expect(ReportingException.class);
```

Comment:
In order to get the parameter and determine whether it's a image, we first have to upload and process the jasper template